package.json

```diff
@@ -58,10 +58,14 @@
     "test:evals:llm": "pnpm --filter ./packages/evals test src/metrics/llm",
     "test:observability": "pnpm --filter ./observability/* test",
     "lint-staged": "lint-staged",
+    "lint": "turbo lint --filter \"./packages/**\" --filter \"./stores/**\" --filter \"./auth/**\" --filter \"./voice/**\" --filter \"./deployers/**\" --filter \"./workflows/**\" --filter \"./observability/**\" --filter \"./client-sdks/**\" --filter \"!@mastra/playground\"",
     "preinstall": "npx only-allow pnpm",
     "prepare": "husky",
-    "prettier:format": "prettier --write .",
+    "prettier:format": "prettier --write . --log-level warn",
     "prettier:format:check": "prettier --check .",
+    "list-changed-files": "(git diff --name-only --diff-filter=ACMR $(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')...HEAD && git diff --name-only --diff-filter=ACMR && git diff --cached --name-only --diff-filter=ACMR) | sort -u",
```

Comment:
not sure if I trust this :D It takes about 44s in CI, not sure if it's worth the optimisation

Reply:
that's fair :) mostly useful locally where it takes half a second instead of 20